sx/pisa3/pisa_util.py

```diff
@@ -470,7 +468,7 @@ def __getattr__(self, name):
                      % (self.__class__.__name__, name)
             raise AttributeError(e)
 
-_rx_datauri = re.compile("^data:(?P<mime>[a-z]+/[a-z]+);base64,(?P<data>.*)$", re.M | re.DOTALL)
```

Comment:
I'm not sure I like the removal of `re.MULTILINE` from the flags in the same change. Is there any particular reason for it? Base64 encoded data URIs are allowed to contain whitespace, so I think it makes more sense to leave this in.